bin/generate.py

```diff
@@ -273,19 +269,19 @@ class Config:
     # Used at each directory or testcase level.
 
     def parse_solution(p, x, path):
-        check_type('Solution', x, [None, str], path)
+        check_type('Solution', x, [type(None), str], path)
```

Comment:
Unforunately not, no :upside_down_face: See the commit message of ce0bcd598bdf38dda8f069b21e8ad938959f40c8:

> Note that you cannot directly access `NoneType` in Python 3, so instead we need to write `type(None)`.
> This makes the `types` list more consistent as well, because all elements are now actually types (whereas `None` is a value, not a type).

I guess we could write `NoneType = type(None)` at the top of the file but that's also not too pretty :joy: